source/user-manual/reference/ossec-conf/vuln-detector.rst

```diff
@@ -23,15 +23,19 @@ Options
    :depth: 1
    :backlinks: none
 
-+---------------------------+------------------------------------+
-| Options                   | Allowed values                     |
-+===========================+====================================+
-| `enabled`_                | ``yes``, ``no``                    |
-+---------------------------+------------------------------------+
-| `feed-update-interval`_   | Positive number + Time unit suffix |
-+---------------------------+------------------------------------+
-| `index-status`_           | ``yes``, ``no``                    |
-+---------------------------+------------------------------------+
++---------------------------+--------------------------------------+
+| Options                   | Allowed values                       |
++===========================+======================================+
+| `enabled`_                | ``yes``, ``no``                      |
++---------------------------+--------------------------------------+
+| `feed-update-interval`_   | Positive number + Time unit suffix   |
++---------------------------+--------------------------------------+
+| `index-status`_           | ``yes``, ``no``                      |
++---------------------------+--------------------------------------+
+| `offline-url`_            | ``<path>`` (starting with \file://)  |
```

Comment:
We could use `file://<LOCAL_FILE_PATH>` and `http[s]://<URL>` instead

Reply:
Good catch!